portal-api/src/v1/validation/loan-rules/facility-stage-unconditional/requested-cover-start-date.js

```diff
@@ -30,7 +30,7 @@ module.exports = (submittedValues, deal, errorList) => {
     if (!dealHasBeenSubmitted) {
       if (moment(requestedCoverStartDateTimestamp).isBefore(nowDate)) {
         newErrorList.requestedCoverStartDate = {
-          text: 'Requested Cover Start Date must be today or in the future',
+          text: 'Requested Cover Start Date must be on the application submission date or in the future',
```

Comment:
should this be
`Requested Cover Start Date must be the same as the application submission date or in the future`
?

Reply:
The one I set is fine with Clara when I checked.  Can change if required?

Reply:
nop. if she's happy, then I'm happy :)